### PR TITLE
fix(test) : Planner EE tests for adding a new area and collaborator and assigning it to a workitem

### PR DESCRIPTION
**Pull request by Raunak1203**
This PR adds tests for adding New area and collaborator and assigning it to a workitem

### CODE REVIEW

File: ee_tests/src/page_objects/space_settings.page.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { browser, element, by, By, ExpectedConditions as until, $, $$, ElementFinder } from 'protractor';
+import * as support from '../support';
+import { TextInput, Button, BaseElement, BaseElementArray, Clickable } from '../ui';
+import { BasePage } from '.';
+
+
+export class SpaceSettings extends BasePage {
+  settings = new BaseElement($('.pficon-settings'), 'Settings');
+  areasTab = new Button (element(by.xpath('//span[contains(text(),\'Areas\')]')), 'Areas');
```
Comment (ppitonak) [inline ee_tests/src/page_objects/space_settings.page.ts]: Please avoid xpath wherever possible, use `by.cssContainingText()` instead

Comment (Raunak1203) [reply ee_tests/src/page_objects/space_settings.page.ts]: @ppitonak it is for Space settings (adding areas and Collaborators to space) and then added areas and collaborators can be assigned to a workitem.

**Comment by Raunak1203** [comment]
Sure @rgarg1

**Comment by Raunak1203** [comment]
@ppitonakI've updated selectors. and one question do I need to add logout() in afterAll() . How are we running tests in CICO ?

File: ee_tests/src/page_objects/space_settings.page.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { browser, element, by, By, ExpectedConditions as until, $, $$, ElementFinder, Key } from 'protractor';
```
Comment (jarifibrahim) [inline ee_tests/src/page_objects/space_settings.page.ts]: All the imports are not being used.

Comment (Raunak1203) [reply ee_tests/src/page_objects/space_settings.page.ts]: @jarifibrahim done

**Comment by Raunak1203** [comment]
Closing this PR , will be opening a new PR

**Pull request closed without merging**